cli/src/main.rs

```diff
@@ -261,7 +261,9 @@ fn main() {
     let parsed_data = glob_paths
         .par_iter()
         .map(|filepath| {
-            let data = std::fs::read_to_string(filepath).unwrap();
+            let data = std::fs::read_to_string(filepath)
+                .map_err(|e| format!("{filepath}, {e}"))
+                .unwrap();
```

Comment:
Instead of unwrapping a string, recommend just `unwrap_or_else` with a `panic`:

```rust
std::fs::read_to_string(filepath)
    .unwrap_or_else(|e| panic!("failed to read file at {filepath:?}: {e}"))
```